common/formatting/line_wrap_searcher.h

```diff
@@ -46,7 +46,9 @@ void DisplayEquallyOptimalWrappings(
 // Returns false as soon as calculated line length exceeds maximum, or a token
 // that requires a newline is encountered.  If everything fits, then return
 // true.
-bool FitsOnLine(const UnwrappedLine& uwline, const BasicFormatStyle& style);
+bool FitsOnLine(const UnwrappedLine& uwline,
+                const BasicFormatStyle& style,
+                int *current_column = NULL);
```

Comment:
s|NULL|nullptr|

What is this new `current_column` used for?  (Explain in prototype comment.)

Reply:
Done